### ISSUE TITLE
Potential crash from not having env in get_config

### ISSUE THREAD

**Issue by bjornsnoen** [thenbe/neotest-playwright#11]
The env setting defined in adapter settings is not used when running the get_config function in playwright.ts. This means that if any env vars are required inside the detected playwright conf file, there's a potential for a crash. Minimal reproduction would be the following, using `tiny-invariant` in the project.

```lua
-- neotest.lua
local neotest = require('neotest')
neotest.setup({
    require('neotest-playwright').adapter({
        options = {
            persist_project_selection = true,
            enable_dynamic_test_discovery = true,
            cwd = function(_)
               return vim.fn.getcwd()
            end,
            env = {
                NEXT_PUBLIC_SITE_URL = 'http://somesite.localhost',
            },
        },
    })
})
```

```ts
// playwright.config.ts
import invariant from 'tiny-invariant'

invariant(process.env.NEXT_PUBLIC_SITE_URL != null, "Oop big error")
```

Try to run a test

**Comment by thenbe** [comment]
Thank you for the report. Can you confirm whether this is fixed for you in #12?

```diff
-	"thenbe/neotest-playwright",
+	{
+		"thenbe/neotest-playwright",
+		branch = "env-in-config",
+	},
```

**Comment by bjornsnoen** [comment]
Yep, that fixes it for me
